test/project-config/translations/messages-en.properties

```diff
@@ -330,3 +330,11 @@ contact.child.mother.indicator.potable.water = Number of households with access
 contact.child.mother.indicator.cash.transfert = Number of households that have benefited from cash transfers
 contact.child.mother.indicator.sexual.mutilation = Number of children with sexual mutilations
 contact.child.mother.indicator.malnutrition = Number of children aged 6-59 months having been screened for malnutrition in a household
+
```

Comment:
I think we should not commit this. This is the result of add the stock out featureWe should revert to the original file after each test